com.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: cpu-x
+
+finish-args:
```

Comment:
there should be a `device=dri` for GL

Reply:
Nice catch. Added in 06efb87b39ef88747efd0d3a7c93b9177d289af8.